### PR TITLE
Check language plugins in about tests

### PR DESCRIPTION
**Pull request by Frassle**
Before I go and break something here, adding some tests to check that "about" reports back the language plugin used and the version.

### CODE REVIEW

**Review by lunaris — APPROVED** [review]

**Comment by Frassle** [comment]
Oh that's odd. These pass on my local builds, what's CI doing different.

**Comment by Frassle** [comment]
Turns out we were inconsistent about which Version variable we used and set. Second commit here deletes the one from pkg and updates everything to use the sdk one.

File: Makefile [hunk 1]
```diff
@@ -71,21 +71,21 @@ generate::
 	echo "This command does not do anything anymore. It will be removed in a future version."
 
 build:: build_proto build_display_wasm go.ensure
-	cd pkg && go install -ldflags "-X github.com/pulumi/pulumi/pkg/v3/version.Version=${VERSION}" ${PROJECT}
+	cd pkg && go install -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ${PROJECT}
 
 build_display_wasm:: go.ensure
-	cd pkg && GOOS=js GOARCH=wasm go build -o ../bin/pulumi-display.wasm -ldflags "-X github.com/pulumi/pulumi/pkg/v3/version.Version=${VERSION}" ./backend/display/wasm
+	cd pkg && GOOS=js GOARCH=wasm go build -o ../bin/pulumi-display.wasm -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common//version.Version=${VERSION}" ./backend/display/wasm
```
Comment (justinvp) [inline Makefile:77]:
```suggestion
	cd pkg && GOOS=js GOARCH=wasm go build -o ../bin/pulumi-display.wasm -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ./backend/display/wasm
```

File: Makefile [hunk 2]
```diff
@@ -71,21 +71,21 @@ generate::
...
 
 install:: .ensure.phony go.ensure
-	cd pkg && GOBIN=$(PULUMI_BIN) go install -ldflags "-X github.com/pulumi/pulumi/pkg/v3/version.Version=${VERSION}" ${PROJECT}
+	cd pkg && GOBIN=$(PULUMI_BIN) go install -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common//version.Version=${VERSION}" ${PROJECT}
```
Comment (justinvp) [inline Makefile:80]:
```suggestion
	cd pkg && GOBIN=$(PULUMI_BIN) go install -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ${PROJECT}
```

**Review by justinvp — APPROVED** [review]
A couple typos, but otherwise LGTM

**Pull request merged**